src/include/cache_inline.h

```diff
@@ -87,9 +88,9 @@ __wt_cache_stuck(WT_SESSION_IMPL *session)
     WT_CACHE *cache;
 
     cache = S2C(session)->cache;
-    WT_ASSERT(session, cache->evict_aggressive_score <= WT_EVICT_SCORE_MAX);
```

Comment:
Instead of doing two atomic loads, I'd pull this out into its own local variable.